index.html

```diff
@@ -13,47 +13,60 @@ <h1>ColoRandom</h1>
         <form class="box-view">
 
           <div class="unit" id="unit1">
-            <div class="palette-box"></div>
-            <p class="hex">
+            <div class="color-field"></div>
+            <p class="hex1">
               <img class="unlock" src="./lock-open-solid.svg" alt="Unlocked" />
             </p>
             <svg class="lock"></svg>
           </div>
 
           <div class="unit" id="unit2">
-            <div class="palette-box"></div>
-            <p class="hex">
+            <div class="color-field"></div>
```

Comment:
This is more clear. I will spend some time trying to adjust the html for layout purposes .